Src/PerformanceCollector/Shared/Implementation/PerformanceCounterUtility.cs

```diff
@@ -272,6 +272,12 @@ internal static IList<string> GetClrProcessInstances()
             }
 
             usesPlaceholder = true;
+
```

Comment:
why this method was not moved to "StandardPerfCollector" folder. It doesn't make sense to have it in the common folder